doc/holochain_101/src/conductor_instances.md

```diff
@@ -19,10 +19,10 @@ A reference to the given ID of a defined [DNA](./conductor_dnas.md)
 A table for configuring the approach to storage of the local source chain and DHT for this instance
 
 #### `StorageConfiguration.type`: `enum`
-Select between different storage implementations. There are two so far:
+Select between different storage implementations. There are three so far:
 - `memory`: Persist actions taken in this instance only to memory. Everything will disappear when the Conductor process stops.
 - `file`: Persist actions taken in this instance to the disk of the device the Conductor is running on. If the Conductor process stops and then restarts, the actions taken will resume at the place in the local source chain they last were at.
-- `pickle` : Persists to a fast rust implementation. This persists every 5 minutes and on a drop trait, actions will continue from where they were last were at if application is every restarted. Takes in path string which is directory where pickledb instance will be persisting to
+- `pickle` : Persists to a fast memory call which is eventually persisted to a file storage every 5 seconds. The actions taken will also resume at the place in the local source chain they were last. It implements a drop trait which allows data to be flushed out whenever an application error occurs.
```

Comment:
```suggestion
- `pickle` : Persists to a fast memory call which is eventually persisted to a file storage every 5 seconds. The actions taken will also resume at the place in the local source chain they were last. If an application error does occur, it will make sure to persist the latest data prior to any shutdown occuring.
```